addons/account/models/partner.py

```diff
@@ -295,6 +295,13 @@ class ResPartner(models.Model):
     _name = 'res.partner'
     _inherit = 'res.partner'
 
+    current_company_country_code = fields.Many2one(string="Current Company Country", comodel_name='res.country', compute='_compute_current_company_country_code')
```

Comment:
@d-fence @KangOl Can we have an exception for a teeny new computed field in stable?

We also have a new module, not seen by runbot (`l10n*`), not sure if we need to declare it in the upgrade somehow?

Reply:
@d-fence @KangOl Could you please have a look when you have a chance ?

Reply:
@d-fence @KangOl any chance we can have a look at this ? We need to have this done to work on the forward port and this will require some work.